backend/vkt/src/main/java/fi/oph/vkt/config/AppConfig.java

```diff
@@ -38,6 +38,7 @@ public EmailSender emailSender(@Value("${app.email.service-url}") String emailSe
   @Bean
   public CasTicketValidator casTicketValidator(@Value("${app.cas.validate-ticket-url}") String casValidateUrl) {
```

Comment:
`app.cas.validate-ticket-url`:in osalta default arvo voisi ehkä olla untuvan vastaava url mut tuolla nyt ei suurta väliä. Tälle tosin voisi lisätä sopivan arvon `vkt.properties.template`:n alle kun sellainen sieltä puuttuu.

Reply:
Jees, muutin nimen ja lisäsin arvon `vkt.properties.template`.

app.auth.service-url kuvastaa sitä sourcea, mistä requesti on lähtenyt ja sen pitää matchata sitä jota vasten ticketin validointi tehdään